pages/dkp/kaptain/2.0.0/operations/mysql-cluster/index.md

```diff
@@ -179,13 +206,13 @@ The operation will use the latest backup data created by the backup referenced i
 To restore from a specific backup location, set the `BACKUP_RESTORE_SOURCE` variable to point to the location of the backup you want to use to restore your cluster.
 ```bash
 # Full path to the backup folder, for example, s3://mysql-store-backup/kaptain-mysql-store-2021-04-05-21:47:24-full
-export BACKUP_RESTORE_SOURCE=""
+BACKUP_RESTORE_SOURCE=""
```

Comment:
I'm not sure why we had these exports here; they are not needed.

Reply:
The difference between "export" vs "no export" is explained here:
https://www.baeldung.com/linux/bash-variables-export
We do not know exact use cases and scenarios and I think keeping export is safer. It is also what we do in other parts of our documentation so we should keep it uniform

Reply:
Having done a fair amount of shell scripting in my days, I can safely say that in these instructions, there is absolutely no reason to export them. Personally, it feels messy to export variables unnecessarily. To me, it's a bit like adding an extra step in documentation that does absolutely nothing to achieve your end goal.

However, it is relatively harmless here so I can understand a desire to keep a consistent style. This isn't a hill I will choose to die on.